doc/nmstate-autoconf.8.in

```diff
@@ -0,0 +1,68 @@
+.\" Manpage for nmstate-autoconf.
+.TH nmstate-autoconf 8 "@DATE@" "@VERSION@" "nmstate-autoconf man page"
+.SH NAME
+nmstate-autoconf \- A nmstate command line tool to automatically configure the
+network state using LLDP information
+.SH SYNOPSIS
+.B nmstate-autoconf \fR[\fIINTERFACE_NAME\fR] [\fB--bond-vlans\fR]
+.br
+.SH DESCRIPTION
+.B nmstate-autoconf\fR is a command line tool that uses \fIlibnmstate\fR in order to
+confgure the network state automatically using LLDP information. This tool is
+experimental only.
+.PP
+For multiple interface names, use comma to separate them. You can also use
+patterns for interface names:
+.RS
+.B *\fR matches everything
+.br
+.B ?\fR matches any single character
+.br
+.B [seq]\fR matches any character in seq
+.br
+.B [!seq]\fR matches any character not in seq
+.RE
+.PP
+For example, to show all interfaces starts with eth:
```

Comment:
not `show all interfaces`, but `enable LLDP auto configuration on all interfaces`.